### PR TITLE
[Editor] Add some resizers all around an editor (bug 1843302)

### PR DESCRIPTION
**Pull request by calixteman**
- it'll improve the way to resize images: diagonally (in keeping ratio between dimensions) or horizontally/vertically.
- the resizer was almost invisible in HCM.
- make a resize undoable/redoable.

### CODE REVIEW

**Review by Snuffleupagus — COMMENTED** [review]
Does this also effectively "fix" https://bugzilla.mozilla.org/show_bug.cgi?id=1776906 by making it obsolete?

File: src/display/editor/editor.js [hunk 1]
```diff
@@ -401,6 +401,295 @@ class AnnotationEditor {
...
+    const resizingClassName = `resizing${name
+      .charAt(0)
+      .toUpperCase()}${name.slice(1)}`;
+    this.parent.div.classList.add(resizingClassName);
+    const pointerMoveOptions = { passive: true, capture: true };
+    window.addEventListener(
+      "pointermove",
+      boundResizerPointermove,
+      pointerMoveOptions
+    );
```
[the remark this answers is not in the document]
Comment (calixteman) [reply src/display/editor/editor.js:449]: The same options are required when removing the listener.
So if we change something in these options, then we won't have to change them in two different places (and potentially forget one place).

**Comment by calixteman** [comment]
> Does this also effectively "fix" https://bugzilla.mozilla.org/show_bug.cgi?id=1776906 by making it obsolete?

Yes and we should move this bug somewhere else.

File: web/annotation_editor_layer_builder.css [hunk 2]
```diff
@@ -189,11 +183,108 @@
...
+  height: 100%;
 }
 
-.annotationEditorLayer .stampEditor canvas {
+.annotationEditorLayer .resizers {
   width: 100%;
   height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
```
Comment (Snuffleupagus) [inline web/annotation_editor_layer_builder.css:196]: Nit: Could we achieve the same thing the following way, or does that not work for reasons I'm overlooking?
```suggestion
  inset: 0;
  position: absolute;
```

**Review by Snuffleupagus — APPROVED** [review]
It's slightly unfortunate that "shortcomings" in the web-platform forces us to implement this, however it does seem to work well in my testing :-)

r=me, thank you!

**Pull request merged**